apps/dashboard/test/fixtures/config/ondemand.d/booleans.yml

```diff
@@ -5,6 +5,7 @@ bc_simple_auto_accounts: true
 per_cluster_dataroot: true
 file_navigator: true
 remote_files_enabled: true
+remote_validation: true
```

Comment:
OK - let's keep the option but can we rename it to `remote_files_validation`?

Reply:
Renamed.